### DISCUSSION TITLE
Unable to export logs to --log-file

### DISCUSSION THREAD

**Discussion by tashrifbillah** [pytest-dev/pytest#8228] [Q&A]
Hi all,

I am new to pytest and trying to export test logs to a file. Here is my setup:

test_one.py:

```
def test_equal():
    assert 1==1
```

> pytest --log-file log.txt test_one.txt

Although I have specified a `--log-file`, all the following logs are on the console and don't get exported to `log.txt`:
```vim
$ pytest test_one.py
======================== test session starts ========================
platform linux -- Python 3.6.12, pytest-6.2.1, py-1.10.0, pluggy-0.13.1
rootdir: /tmp
plugins: dash-1.18.1
collected 1 item

test_one.py .                                                 [100%]
```

What am I missing?

**Comment by RonnyPfannschmidt** [comment]
logs refers to the data from the python stdlib logging library, not the general cli output of pytest

so if you added something like

```
import logging
log = logging.getLogger(__name__)

# and in the test
log.warning("im here")

```
you would see that message logged in the logfile
